### DISCUSSION TITLE
Add gene labels on Volcano markerPlot

### DISCUSSION THREAD

**Discussion by tzhu-bio** [GreenleafLab/ArchR#1048] [General]
Is there an easy way to add gene labels on Volcano markerPlot?

**Comment by rcorces** [comment]
We do not provide code to do this. you would have to do this manually

**Reply by NoemieL** [reply]
Hi, I am also interested to do this. Where or how could we find/get the data (gene list of the colored dots)to do it? Thanks

**Reply by rcorces** [reply]
from the SummarizedExperiment objects output by various ArchR functions such as getMarkerFeatures()

**Reply by NoemieL** [reply]
Thanks. I did not get the name of the gene when I used getMarker() after getMarkerFeatures(). Do you have any idea why? 

markerTest_C11_C12<- getMarkerFeatures(
  ArchRProj = ArchRProj, 
  useMatrix = "PeakMatrix",
  groupBy = "Clusters2",
  testMethod = "wilcoxon",
  bias = c("TSSEnrichment", "log10(nFrags)"),
  useGroups = "C11",
  bgdGroups = "C12"
)

getMarkers(markerTest_C11_C12, cutOff = "FDR <= 0.05 & Log2FC >= 1")$C11
DataFrame with 6515 rows and 7 columns
       seqnames     idx     start       end    Log2FC         FDR  MeanDiff
          <Rle> <array>   <array>   <array> <numeric>   <numeric> <numeric>
74122     chr19     505   1607211   1607711   3.99734 2.55652e-30  0.292120
25173     chr11    1043   8634439   8634939   4.43359 1.35060e-24  0.263550
137311     chr6    6743 116631404 116631904   4.03947 6.72648e-22  0.210086
148203     chr8     345  11429851  11430351   3.12054 1.82864e-19  0.171053
18771     chr10    2244  34674123  34674623   3.33251 1.23828e-18  0.187013
...         ...     ...       ...       ...       ...         ...       ...
43283     chr13     586  32414952  32415452   2.99563   0.0499483 0.0274050
47838     chr14    2183  64976615  64977115   2.64981   0.0499483 0.0171076
151266     chr8    3408  69858608  69859108   2.69518   0.0499483 0.0189222
31704     chr11    7574 109155211 109155711   1.26343   0.0499497 0.0413292
29736     chr11    5606  71120524  71121024   1.60546   0.0499748 0.0403996

**Reply by rcorces** [reply]
the `idx` column should match the index from your `peakSet` which contains the name of the closest gene. Otherwise you would map to peakToGeneLinks or coaccessibility
